gwpy/astro/range.py

```diff
@@ -75,9 +101,31 @@ def decorated_func(psd, *args, **kwargs):
     return decorated_func
 
 
+def _copy_docstring(func):
+    def decorated_func(target):
+        import warnings
+        warnings.warn(
+            "this estimator currently uses the LIGO SenseMon (LIGO-T030276), "
+            "which will be replaced by a model with cosmological corrections "
+            "in gwpy-2.2.0.\n\n"
+            "The proper SenseMon method is now gwpy.astro.{}".format(
+                func.__name__),
+            DeprecationWarning,
+        )
+        target.__doc__ = func.__doc__
+        return target
+    return decorated_func
```

Comment:
This looks like a reimplementation of the `gwpy.utils.decorators.deprecated_function` decorator, please consider using that instead.

Reply:
Aha! This has been addressed, #1294 will need to go in first